doc/man/bitcoin-qt.1

```diff
@@ -274,10 +274,6 @@ Tor control port to use if onion listening enabled (default:
 .IP
 Tor control port password (default: empty)
 .HP
-\fB\-upnp\fR
-.IP
-Use UPnP to map the listening port (default: 0)
-.HP
```

Comment:
why is this removed?

Reply:
Thanks, that removal was accidental, as miniupnpc wasn't available when i compiled the 0.17.2 binaries. Fixed up now.